mne/viz/backends/_pysurfer_mayavi.py

```diff
@@ -0,0 +1,433 @@
+            mesh, contours=contours, line_width=1.0, vmin=vmin, vmax=vmax,
+            opacity=opacity, figure=self.fig)
+        cont.module_manager.scalar_lut_manager.lut.table = colormap
+
+    def surface(self, surface, color=(0.7, 0.7, 0.7), opacity=1.0,
+                vmin=None, vmax=None, colormap=None,
+                backface_culling=False):
+        """Add a surface in the scene.
+
+        Parameters
+        ----------
+        surface: surface object
+            The information describing the surface.
+        color: (red, green, blue)
+            The color of the surface.
+        opacity: float
+            The opacity of the surface.
+        vmin: float | None
+            vmin is used to scale the colormap.
+            If None, the min of the data will be used
+        vmax: float | None
+            vmax is used to scale the colormap.
+            If None, the max of the data will be used
+        colormap:
+            The colormap to use.
+        backface_culling: bool
+            If True, enable backface culling on the surface.
+        """
+        # Make a solid surface
+        mesh = _create_mesh_surf(surface, self.fig)
+        surface = self.mlab.pipeline.surface(
+            mesh, color=color, opacity=opacity, vmin=vmin, vmax=vmax,
+            figure=self.fig)
+        if colormap is not None:
+            surface.module_manager.scalar_lut_manager.lut.table = colormap
+        surface.actor.property.backface_culling = backface_culling
+
+    def sphere(self, center, color, scale, opacity=1.0,
+               backface_culling=False):
+        """Add sphere in the scene.
+
+        Parameters
+        ----------
+        center: ndarray, shape(n_center, 3)
+            The list of centers to use for the sphere(s).
+        color: (red, green, blue)
+            The color of the sphere(s).
+        scale: float
+            The scale of the sphere(s).
+        opacity: float
+            The opacity of the sphere(s).
+        backface_culling: bool
+            If True, enable backface culling on the sphere(s).
+        """
+        if center.ndim == 1:
+            x = center[0]
+            y = center[1]
+            z = center[2]
+        elif center.ndim == 2:
+            x = center[:, 0]
+            y = center[:, 1]
+            z = center[:, 2]
+        surface = self.mlab.points3d(x, y, z, color=color,
+                                     scale_factor=scale, opacity=opacity,
+                                     figure=self.fig)
+        surface.actor.property.backface_culling = backface_culling
+
+    def quiver3d(self, x, y, z, u, v, w, color, scale, mode, resolution=8,
+                 glyph_height=None, glyph_center=None, glyph_resolution=None,
+                 opacity=1.0, scale_mode='none', scalars=None,
+                 backface_culling=False):
+        """Add quiver3d in the scene.
+
+        Parameters
+        ----------
+        x: array
```

Comment:
it's a good practive to document the shape eg

`x : array, shape (n_quivers,)`